### ISSUE TITLE
BUG - 2 - All - Accessibility: Labs & Tests Report section is read as large block of text, cannot navigate within it

### ISSUE THREAD

**Issue by SKMOstudio** [department-of-veterans-affairs/va-mobile-app#11996]
## What happened?

When an accessibility user is on the Detail page for a Lab/Test report, the 'Report' section of the page is read out as a large block of text. This makes it difficult for screen reader users to navigate and parse the large text of information within this sesction.

## Specs:

- Device: Android/iOS
- OS: Latest
- User Account: Demo
- Accessibility State: Screen reader ON, default settings.

## Steps to Reproduce

1. As a demo user or a staging user, ensure the screen reader is ON, then access the Lab & Tests access.
2. On the Labs and Tests page, tap on a report to go to the Details page.
3. Navigate down the Details page. Observe - The first few lines of the report can be easily swiped through. When the user reaches the Report section, the content of the section is read out as a large block of text. The user cannot tap or swipe-navigate to focus on specific lines or sections within the report.

## Acceptance Criteria

When a screen reader user focuses on the Report section on a Detail page, they should be able to focus on content within the Report section on a line-by-line basis as best as possible. The user should not have to sit through a long read-through of content to find specific information.

## Bug Severity - BE SURE TO ADD THE SEVERITY LABEL
 
See [Bug Tracking](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/QA#issue-severity) for details on severity levels

- Impact: High **Low** 
- Frequency: **High** Low 

- 3 - Low
- 2 - **High**
- 1 - Critical

## Linked to Story

#11945 

## Screen shot(s) and additional information

<details>
  <summary>Full JSON response for services related to issue (expand/collapse)</summary>
  

</details>

## Ticket Checklist
- [ ] Steps to reproduce are defined
- [ ] Desired behavior is added
- [ ] Labels added (front-end, back-end, global, Health, relevant feature, accessibility, etc)
- [ ] Estimate of 1 added (for front-end tickets)
- [ ] Added either to the relevant feature epic (if found during new feature implementation), or the relevant team's bug epic ([Global](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8302), [Health & Benefits](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8303), [API](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8304), [QART](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8305))
